### DISCUSSION TITLE
关于chartero的一些小建议

### DISCUSSION THREAD

**Discussion by tina140119** [volatile-static/Chartero#102] [Ideas]
先祝作者大大新的一年身体健康，万事如意！！

非常喜欢chartero这款插件，与此同时也想提供一些小小的建议让chartero更加优秀。

1. 首先这个阅读热图的功能我非常的喜欢，但是是否可以考虑再出一个柱状图统计一周7天每天的阅读时长？
![image](https://github.com/volatile-static/Chartero/assets/115853966/0eb5d05b-a4fd-4c94-9c67-cb8a20f6ced8)
就比如下面这种：
![image](https://github.com/volatile-static/Chartero/assets/115853966/eb25bace-3c5f-4737-8c92-7282e8c55425)
横坐标为周一到周天，纵坐标为时间，可以让数据更加直观。
2. 其次，我很想知道自己每天阅读的工作时长，是否考虑出一个视图或者就一个文字版展示在首页提示我今天阅读时长
![image](https://github.com/volatile-static/Chartero/assets/115853966/19b20e1f-02ed-47db-a9ea-c9f7825ef7c1)
比如就放在这个试图板块处，或者其他一眼能看到的地方，不需要我再次点击进入才能看到

**再次感谢作者UP~**

**Comment by volatile-static** [comment]
感谢支持！很好的建议，但设计上有些复杂，或许之后有时间后再实现。近期的计划是支持附件格式存储阅读记录。

**Reply by tina140119** [reply]
好的，蟹蟹大大！祝一切顺利哦！

**Comment by simateimourian** [comment]
Could you please share a source for using this plugin? I installed it and categorized my articles but in the dashboard I can not see anything.
